python/dolfinx/wrappers/graph.cpp

```diff
@@ -68,6 +68,9 @@ void graph(py::module& m)
           "Links (edges) of a node",
           py::return_value_policy::reference_internal)
       .def("array", &dolfinx::graph::AdjacencyList<std::int64_t>::array)
+      .def("offsets", &dolfinx::graph::AdjacencyList<std::int64_t>::offsets,
+           "Index to each node in the links array",
+           py::return_value_policy::reference_internal)
```

Comment:
I wonder if the return value policy is required for Eigen, or if pybind11 does this automatically? Could we test?